File: dart/dart-iterate-map/example2.dart
// https://codevscolor.com/dart-iterate-map
main() {
  var numMap = Map();
  numMap["one"] = 1;
  numMap["two"] = 2;
  numMap["three"] = 3;
  numMap["four"] = 4;
  numMap["five"] = 5;

  for (var k in numMap.keys) {
    print("Key : $k, value : ${numMap[k]}");
  }
}

File: kotlin/kotlin-sort-map-by-value/example2.kt
// https://codevscolor.com/kotlin-sort-map-by-value
fun main() {
    val givenMap = hashMapOf<String, Int>()
    givenMap["one"] = 1
    givenMap["two"] = 2
    givenMap["three"] = 3
    givenMap["four"] = 4
    givenMap["five"] = 5
    givenMap["six"] = 6

    println("Given map :")
    givenMap.forEach { (k, v) -> println("$k => $v") }

    var sortedMap: MutableMap<String,Int> = LinkedHashMap()

    givenMap.entries.sortedBy { it.value }.forEach{sortedMap[it.key] = it.value}

    println("Sorted map :")
    sortedMap.forEach { (k, v) -> println("$k => $v") }
}